tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects.robot

```diff
@@ -0,0 +1,439 @@
+    Workbench Should Be Listed      workbench_title=${WORKBENCH_3_TITLE}
+    Reload Page
+    Wait Until Project Is Open    project_title=${PRJ_TITLE}
+    Workbench Status Should Be      workbench_title=${WORKBENCH_3_TITLE}      status=${WORKBENCH_STATUS_STARTING}
+    Run Keyword And Continue On Failure    Wait Until Workbench Is Started     workbench_title=${WORKBENCH_3_TITLE}
+    Check Corresponding Notebook CR Exists      workbench_title=${WORKBENCH_3_TITLE}   namespace=${ns_name}
+    Reload Page
+    Wait Until Project Is Open    project_title=${PRJ_TITLE}
+    ${connected_woksps}=    Create List    ${WORKBENCH_3_TITLE}
+    Storage Should Be Listed    name=${pv_name}    description=${PV_DESCRIPTION}
+    ...                         type=Persistent storage    connected_workbench=${connected_woksps}
+    Storage Size Should Be    name=${pv_name}    namespace=${ns_name}  size=${PV_SIZE}
+
+Verify User Can Stop A Workbench
+    [Tags]    Sanity    ODS-1817
+    [Documentation]    Verifies users can stop a running workbench from project details page
+    Open Data Science Project Details Page       project_title=${PRJ_TITLE}
+    Stop Workbench    workbench_title=${WORKBENCH_TITLE}    press_cancel=${TRUE}
+    Stop Workbench    workbench_title=${WORKBENCH_TITLE}
+    # add checks on notebook pod is terminated but CR is present
+
+Verify User Can Launch A Workbench
+    [Tags]    Sanity    ODS-1815
+    [Documentation]    Verifies users can launch/open a running workbench from project details page
+    Open Data Science Projects Home Page
+    ${ns_name}=    Get Openshift Namespace From Data Science Project   project_title=${PRJ_TITLE}
+    Open Data Science Project Details Page       project_title=${PRJ_TITLE}
+    Start Workbench     workbench_title=${WORKBENCH_TITLE}
+    Launch Workbench    workbench_title=${WORKBENCH_TITLE}
+    Check Launched Workbench Is The Correct One     workbench_title=${WORKBENCH_TITLE}     
+    ...    image=${NB_IMAGE}    namespace=${ns_name}
+
+Verify User Can Stop A Workbench From Projects Home Page
+    [Tags]    Sanity    ODS-1823
+    [Documentation]    Verifies users can stop a running workbench from Data Science Projects home page
+    Open Data Science Projects Home Page
+    ${ns_name}=    Get Openshift Namespace From Data Science Project   project_title=${PRJ_TITLE}
+    ${_}    ${workbench_cr_name}=    Get Openshift Notebook CR From Workbench    workbench_title=${WORKBENCH_TITLE}    
+    ...    namespace=${ns_name}
+    Stop Workbench From Projects Home Page     workbench_title=${WORKBENCH_TITLE}   project_title=${PRJ_TITLE}  
+    ...    workbench_cr_name=${workbench_cr_name}    namespace=${ns_name}
+    Workbench Launch Link Should Be Disabled    workbench_title=${WORKBENCH_TITLE}  project_title=${PRJ_TITLE}
+    # add checks on notebook pod is terminated but CR is present
+
+Verify User Can Start And Launch A Workbench From Projects Home Page
+    [Tags]    Sanity    ODS-1818
```

Comment:
I see tests already for `User can Start a Workbench` and `User can Launch a Workbench`. Is an additional test where this is tested again really needed?

Reply:
it's basically testing the same process but done from DS Project home page which is the page where all the projects are listed. 
The other similar tests are starting/launching workbenches from inside the project (i.e., DS Project details page)